README.md

```diff
@@ -36,3 +36,10 @@ The workshop notebooks can also be run on [Sciserver](https://sciserver.org/), w
 - Create a container, give it name, select the *'NAVO-workshop'* image, and click *Create*. Click on the container name to launch the jupyterlab interface.
 
 - The conda environment `navo-env` contains all the packages required to run the notebooks. The notebooks themselves are available under `/home/idies/workspace/navo-workshop`.
```

Comment:
```suggestion
- The conda environment `navo-env` contains all the packages required to run the notebooks. On the file system, the notebooks are available under `/home/idies/workspace/navo-workshop`.

- You can use the Jupyterlab file browser to navigate to the `navo_workshop` then `content` directory, then either the `reference_notebooks` or `use_case_notebooks` subdirectories that contain the notebooks.
```

Reply:
I was just coming out of Dave's update and realized we could give similar guidance here for navigating to the notebooks.